.github/workflows/deploy.yml

```
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - release
```

The release branch will have a date appended to it

```suggestion
      - release-*
```

---

(discussed on slack)